GameServer/Logic/GameObjects/MovementVector.cs

```diff
@@ -34,12 +34,14 @@ public static short FormatCoordinate(float coordinate, float origin)
 
         public static short TargetXToNormalFormat(float value)
         {
-            return FormatCoordinate(value, _game.Map.GetWidth() / 2);
+            var game = Program.ResolveDependency<Game>();
```

Comment:
This is technically still wrong, but alright for this I suppose. We'll know what to fix when the `ResolveDependency` method is removed